app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,8 @@
+class StaticPagesController < ApplicationController
+  def home
```

Comment:
```suggestion
  def home; end
```
tương tự cho các hàm khác